token-service/services/ForecastApiService.js

```diff
@@ -0,0 +1,66 @@
+const { Curl } = require('node-libcurl');
```

Comment:
You could have used axios as well I suppose. It is suitable for node js too as far as I'm concerned

Reply:
Done. I have rewrote methods of token getting with axios lib.